Updating GM Guide and other minor fixes

LockerM opened the pull request:
Major overhaul of GM Guide in progress. Working on following issues: 

- [x] #33 
- [x] #34 
- [x] #61 
- [x] #53 
- [x] #59 (adding in addressing this in the GM Guide, Player Guide Brief and Full) 
- [x] #100 
- [x] #102 
- [x] #73 
- [x] #78 (Partially addressing - renaming to 'Burn Mechanics')
- [x] #69 (Partially addressing - explicit naming of 'Fatigue' as a Status Condition) 

Other minor fixes include:
- Update to CONTRIBUTING doc version reference

CBroz1 commented on 1_Mechanics/02_GMGuide.md:
```diff
@@ -1,82 +1,263 @@
+open to Player questions and ideas, and attempt to work with them to tell the
+story rather than against them. 
+
+You should be an impartial force in the world, controlling both the bad and
+the good without judgement, anger, or aggression. The best sessions feel like
```
```suggestion
You should be an impartial force in the world, controlling both allies and
enemies neutrally. The best sessions feel like
```

CBroz1 commented on 1_Mechanics/02_GMGuide.md:
```diff
@@ -1,82 +1,263 @@
+the good without judgement, anger, or aggression. The best sessions feel like
+everyone is working together to complete an objective, even the GM, rather
+than it feeling like a battle or competition between the Players and the Game
+Master. This is a delicate balance, and GM's are encouraged to ask for their
```
```suggestion
Master. This is a delicate balance, and GMs are encouraged to ask for their
```

CBroz1 commented on 1_Mechanics/02_GMGuide.md:
```diff
@@ -1,82 +1,263 @@
+
+### Game Master Resources
+
+GM's should have several decks of cards available to them during a session so
+as to easily represent multiple groups of Minions and Dealers that the
+Players will fight during the course of an Adventure. At minimum, a GM must
+maintain two separate decks - one for enemy combatants and one for allies of
+the Players. If a GM only has one deck available, they should split the deck
+into two 26 card decks and re-shuffle regularly. 
```
```suggestion
GMs should have at least two decks of cards available during a session 
to represent allies and enemies. More decks may be useful to represent 
different groups. If a GM only has one deck available, they should split the deck
into two 26 card decks and re-shuffle regularly. 
```

CBroz1 commented on 1_Mechanics/02_GMGuide.md:
```diff
@@ -1,82 +1,263 @@
+the Players. If a GM only has one deck available, they should split the deck
+into two 26 card decks and re-shuffle regularly. 
+
+As the Game Master, you do not have to draw a single deck to empty. Unless a
+Dealer you control has an individual deck assigned to them, and they are in
+the active scene for long enough, the GM never invokes Lower Hand by needing
+to reshuffle. If an NPC were to need a mechanical reason to show them being
+exhausted, the same Status Condition can be added to that NPC as if they had
+a level of fatigue.  
```
```suggestion
Most NPCs won't stick around long enough to use a deck completely, and  
therefore won't need to invoke Lower Hand by shuffling. For the
exceptional NPC, try assigning them a unique deck, if available. If 
unavailable, you can decide when to invoke Lower Hand behind the DM screen.
```

CBroz1 commented on 1_Mechanics/02_GMGuide.md:
```diff
@@ -1,82 +1,263 @@
+You can think of a Player's deck of cards like their character's stamina. The
+Adventurers should get progressively more tired throughout the day. This is
+accomplished by having the Players make a series of Checks, which is their
+way of interacting with the world. At lower levels, it is OK for Players not
+to approach the re-draw limit of running out of cards between Rests. However,
+as players progress and level up, the Powers they choose and the Encounters
+they face may test them closer to running out of cards between Rests. 
```
Feature or bug? Shouldn't it be the case that PCs develop more stamina as they level up?

Do we need to start with 52 cards at level1? Would it be better to add the suits as you level up? This would mean that suited checks shouldn't happen below a certain level.

```suggestion
way of interacting with the world. At lower levels, Players may not
to approach the re-draw limit of running out of cards between Rests. However,
as players progress and level up, the Powers they choose and the Encounters
they face may test them closer to running out of cards between Rests. 
```

CBroz1 commented on 1_Mechanics/02_GMGuide.md:
```diff
@@ -1,82 +1,263 @@
+HP and FP are also balanced by Rests. It is expected and recommended that a
+party will take 1-2 Quick Rests between Full Rests during each day of in-game
+time, depending on the Encounters they face. While Resting, the Players will
+be regaining their Health and Fate Points, expending Rest Cards along the
+way. This should be kept in mind when planning Encounters that deal damage,
+since Players will want to take a Quick Rest if their HP or FP drops below a
+threshold they feel comfortable with. 
```
I think you could revise systematically in quasi-second person, as this entire file is directed at GMs. This will cut down on the word count. 

```suggestion
be regaining their Health and Fate Points, and expending Rest Cards. 
Keep this in mind when planning Encounters that deal damage,
since Players will want to take a Quick Rest if their HP or FP drop too low. 
```

CBroz1 commented on 1_Mechanics/02_GMGuide.md:
```diff
@@ -1,82 +1,263 @@
+
+### Working with Character Vulnerabilities
+
+When Players create a character, they choose one or two (or more)
```
I thought the limit was 2 Vulnys. Or is `or more` directed at the possibility of taking more without XP reward?

LockerM replied:
Yea, I put that in there since Players can technically add as many Vulnys as they want. Technically it's up to 3 since they can take 1 Major and 2 Minor, or even 4 if they take 4 Minor, so I figured leaving it open would be best.

CBroz1 commented on 1_Mechanics/02_GMGuide.md:
How much of this is covered already by the full guide? Need it be repeated? I previously had it pretty sparse to only give the crunch

CBroz1 commented on 1_Mechanics/02_GMGuide.md:
```diff
@@ -89,54 +270,425 @@ the probability by as much as 25% for a DR 3 check.
-applies. Attributes are more responsive/reactive, often for Contested Checks. 
-Skills are more active for character-initiated events.
+The below is a list of general information about the Attributes and Skills.
+More information is available in the [Full Players Guide](https://github.com/DeckofAdventures/TheGame/blob/main/1_Mechanics/01_PlayerGuide_Full.md#Characters)
```
```suggestion
More information is available in the 
[Full Players Guide](https://github.com/DeckofAdventures/TheGame/blob/main/1_Mechanics/01_PlayerGuide_Full.md#Characters)
```

CBroz1 commented on 1_Mechanics/02_GMGuide.md:
```diff
@@ -89,54 +270,425 @@ the probability by as much as 25% for a DR 3 check.
+
+Here are some Examples of Skills that are used in Contested Checks, though you
+are free to decide what makes the most sense in a given scenario at your
+table. 1. Tackle Target: Attacker Finesse vs. Target Athletics 2. Convince
+Target, truthfully: Bluffing vs. Target Detection 3. Convince Target,
+deceptively: Bluffing vs. Target Detection
```
```suggestion
table. 

1. Tackle Target: Attacker Finesse vs. Target Athletics 
2. Convince Target, truthfully or deceptively: Bluffing vs. Target Detection 
3. `MISSING`
```

CBroz1 commented on 1_Mechanics/02_GMGuide.md:
```diff
@@ -89,54 +270,425 @@ the probability by as much as 25% for a DR 3 check.
+your narrative to keep the session feeling fresh and lively for all of your
+Players.
+
+For the most updated reference on Epic Events, visit the [Full Player Guide](https://github.com/DeckofAdventures/TheGame/blob/main/1_Mechanics/01_PlayerGuide_Full.md#Epic-Events)
```
I don't want to imply that a doc we write is out of date. The metadata can handle that.

```suggestion
For more information on Epic Events, visit the [Full Player Guide](https://github.com/DeckofAdventures/TheGame/blob/main/1_Mechanics/01_PlayerGuide_Full.md#Epic-Events)
```

CBroz1 reviewed (commented):
Started reviewing, up to gm guide Line 363. Will continue later.

LockerM commented on 1_Mechanics/02_GMGuide.md:
```diff
@@ -1,82 +1,263 @@
+Master is all about in Section 2: “Getting Started as a Game Master”. 
+
+If you or your players have questions or need a helping hand, please reach out
+to the team on [Discord] (https://discord.gg/tmHSdb3TRQ) or via GitHub issues.
```
I created this as a link that never expires in the settings.

CBroz1 replied:
I think all our links should expire eventually. The ones I put up are 50-use. I think "What if 50 ppl joined tomorrow, could we handle it?" ... What if 100?

CBroz1 replied:
also - links can't have a space between text and url value

LockerM commented:
I put some comments in the text to respond to in-line comments but pulling them out here for ease of reference: 

*1. New-GM vs. DofA Specific for Combat GM Guide* (Line 550)
- LOCK: If we keep it all new-GM vs. DofA specific we'll lose some of the 'section' based value of the GM guide. Talking through combat more holistically is helpful here - do we strip it out entirely from DofA specific and make 'Combat' its own main section? 

*2. Why draw new TC during Boss Phases* (Line 637)
- LOCK: I pictured a 5 phase boss fight. The Players have made great progress on the Boss' HP, but if they've been aiming at the same TC for the last 10 rounds of combat they are going to be way less likely to keep hitting. Drawing a new TC for each Phase keeps the odds fresh and prevents a Player from feeling like they 'wasted' a card that would have been a hit during Boss fights

LockerM commented:
Updated GM guide with TOC links, tables, refinements, across the areas that are not commented out in the TOC.

Still more to be done to create a 'final' GM Guide, but the scope of what's covered here should be enough to move us into v1.0.0a4 once merged.

LockerM commented:
Made proposed changes to the language in Player Guides around DR + Mod on Report after playtest feedback. Saving other issue resolution for Player Guides for a future PR that would move us from v1.0.0a4 to v1.0.0b1.

LockerM commented:
Before merge and after approval, let's delete the comments currently in there that were resolved.

CBroz1 reviewed (commented):
Finished review of changes to
1. Player Guide Brief
2. Player Guide Fill
3. Contributing

Larger docs in subsequent review

CBroz1 commented on 1_Mechanics/02_GMGuide.md:
```diff
@@ -7,17 +7,19 @@
 
```
I'm looking at commit 049f2e5. In the future, please separate hard wrap into separate commits that I can skip if I'm reviewing commit-by-commit

LockerM replied:
Will do!

CBroz1 commented on 1_Mechanics/02_GMGuide.md:
```diff
@@ -371,12 +411,17 @@ engage all Players in faster-paced, more free-form roleplaying. Since Players
 
+As the GM, you are actively drawing against the Players on a TC during the
+Epic Event. The odds are skewed in the Players' favor by allowing them to
+draw using Skill Modifiers while the GM draws with a +0 Mod. Your ability to
```
```suggestion
draw using Skill Modifiers while you draw with a +0 Mod. Your ability to 
```

CBroz1 commented on 1_Mechanics/02_GMGuide.md:
```diff
@@ -536,30 +586,32 @@ investigation-based with relatively little combat.
 <!-- BROZ: Do we Bystander distinction? Can't an NPC be either Dealer, Minion
  or Companion, and join players in combat accordingly? -->
 
+<!-- LOCK: Bystander to me is our stand-in for "Allied NPC", whereas a Minion is an enemy term and Companion is a Player Controlled character. I don't like the term "NPC" when talking to GMs who are literally playing the characters, so I feel like using Bystander and Minion/Dealer as GM controlled character terms is more useful here. -->
```
I'd prefer to have a unified term for non-dealer so that their allegiance is ambiguous 
- A: Are they a dealer?
- B: No, they're a `X`

If `X` could be bystander or minion, saying which gives hints at where the narrative 'should' go.

I also don't love the asymmetry between bystander and minion. The former sounds passive (which, if DM-NPC or allied combatant, they're not) and the latter sounds weak ('yeah, you can defo oneshot this one') when that may not be the case at higher levels

LockerM replied:
What term would you suggest in this case? If we have Dealer (for Player or GM controlled), what is a singular term for non-Dealer? 

Minion has been my term for an Enemy non-dealer, where Bystander was a term we had in there for a long time. Minion won't work for townsfolk or shopkeeps, so we'll want to find that separate term that can be all-encompassing.

I'd like to keep "Minion" as a term for a directly controlled Dealer character (Player or GM) such as the Necromancer, where the Skeletons are "Minions" of them rather than independent thinking enemy characters that are non-dealers.

LockerM replied:
Added as Issue #104

CBroz1 commented on 1_Mechanics/02_GMGuide.md:
```diff
@@ -280,8 +299,21 @@ As a Game Master (GM), you will set up various challenges wherein you or a
+One of the initial barriers a new GM must overcome is helping the Players
+understand the Target Card system. As Players get more familiar with the
+system, speed of play increases, and it is then easier for them to understand
+more complex mechanics. Using the [Circular TC-DR Tool](.../1_Mechanics/Circlular_TC-DR_Tool/1_Assembled.PNG) is a helpful visual
```
This might have been my mistake. For relative references
- One dot means 'here'
- Two dots means 'up'
- Slash means folder
```suggestion
more complex mechanics. Using the [Circular TC-DR Tool](./Circlular_TC-DR_Tool/1_Assembled.PNG) is a helpful visual
```

CBroz1 commented on 1_Mechanics/02_GMGuide.md:
```diff
@@ -5,25 +5,25 @@
-    - Skills Breakdown
-    - Contested Checks
-    - Epic Events
+- [Introduction](#Introduction)
```
Did you add these heading links by hand? [This package](https://packagecontrol.io/packages/MarkdownTOC) should be able to auto-add them. It's been a while since there were any updates to [the repo](https://github.com/naokazuterada/MarkdownTOC) so there may be some bugs. If it doesn't work for you, it's pretty quick for me to do

LockerM replied:
Yea I hand coded them so we'd have it in there, good to know there's an auto-add tool we can use in the future, thanks!

CBroz1 reviewed (commented):
Reviewed individual commits for edits related to changes from [fork PR](https://github.com/LockerM/TheGame/pull/1)

Edits on newly added sections to follow

CBroz1 commented:
Regarding the comments on sections for `New` vs. experienced DMs: maybe the current gm guide is `GMGuide_Full` and it could/should be accompanied by `GMGuide_Brief` which is barebones targeted at experienced GMs (who already know how what it means to work with players and balance and encounter). For example
1. Cards not dice
2. Set DR, hit TC. Apply Mod to DR
3. For all values, aim for <10. e.g., split boss health to phases

Just enough to run a level1 session in 30m, Good for words to `ctrl+f` in the full doc

LockerM commented on 1_Mechanics/01_PlayerGuide_Full.md:
```diff
@@ -420,9 +421,9 @@ an Epic Event alone.
-bypassing AP. 
+7. **Poisoned** At the start of their turn, as a Free Action, a Dealer makes a
+DR 3 Vitality Check to shake off the effects of the poison. On a failure,
+they discard a card. The Poisoned Status Condition can stack.
```
In my vision of this, the Poison Condition being able to stack is really what makes it dangerous here. If you get one level, sure discarding one card isn't too bad (plus the forced vitality check, plus if something else forces you to do that check at Lower Hand now it's 3 cards per round even at a 1 stack), but if a Dealer or multiple Minions/Environmental effects cause Poison and now you're at Poison (3) your stamina can start to drain FAST.

CBroz1 replied:
Either way, it needs playtesting. For a playtesting philosophy, it might feel better to start harsher and ease off then start weaker and add

LockerM replied:
Sounds good, I agree. I added it in a commit to start at 2 cards discarded and made it more explicit that each additional stack adds +1 card to the discard.

LockerM commented on CONTRIBUTING.md:
```diff
@@ -48,10 +48,23 @@ Please note that your contributions are shareable and remixable under such licen
+integration, which are modified, adapted, playtested, and ultimately
+integrated into the current or future version.
+
+An Issue will be tagged as "Under Consideration", "In Development", "In
+Playtesting", and "Integrated" to establish where it is in the development
+cycle. Not all ideas will make it to the Integrated step, but every
+Contribution and comment from the Community is valued. 
```
I think if we're leveraging this for public community visibility we'll want something a bit softer than the default "wontfix". The goal here is visibility, and transparency, so if we want to go with closed and not labeled "no thanks" as the way of highlighting "Integrated" that's fine, we'll just want to make sure the community knows this is how it's resolved and indicated what's resolved versus what's being passed over/disregarded. I was envisioning some status to indicate it was merged into the main rules so we can go back to the Issue submitter and say "congrats, your Issue was [INTEGRATED]. You can view the closed issue [here](link)"

CBroz1 replied:
That sounds less like issue labels and more like a public project board where someone can see how their issue is moving though the integration process

LockerM replied:
Hm this might be worth discussing further in that case. Part of why setting it up as Issue labels made sense to me to start is because we will want to be driving community members to engage with Issues - to add comments, create new issues, and keep tabs on outstanding issues they're passionate about or interested in. If we create a public project board it may help for visualizing the development process, but driving people to view that project board feels more passive "hey go to the project board so you can check out what we're working on" versus saying "hey go to Issues, add comments and submit your own so the community can keep updating and refining the game". 

Definitely open to a bigger discussion around how we want to approach this, I think starting with Issue labels and seeing how it goes from there might be a good starting point for heading into Beta, especially given that those issue labels can be folded into a project board for tagging/tracking in the future without us needing to make and maintain a full additional project board at this point when it might not be needed at this stage.

LockerM commented:
> Regarding the comments on sections for `New` vs. experienced DMs: maybe the current gm guide is `GMGuide_Full` and it could/should be accompanied by `GMGuide_Brief` which is barebones targeted at experienced GMs (who already know how what it means to work with players and balance and encounter). For example
> 
> 1. Cards not dice
> 2. Set DR, hit TC. Apply Mod to DR
> 3. For all values, aim for <10. e.g., split boss health to phases
> 
> Just enough to run a level1 session in 30m, Good for words to `ctrl+f` in the full doc

Added Issue #103.

CBroz1 commented on 1_Mechanics/02_GMGuide.md:
Examples below are bold entity name. Change to reflect same here?
```suggestion
**Name**
```

LockerM replied:
I think we can leave it as ##### h5 as originally stated, sorry I didn't update that in the last pass. Making a commit to update that now - I think we should stick with h5 as a naming convention in other docs for NPCs or stat blocks so we can render these specifically uniquely in the HTML/Website render side of things rather than bolding it.

CBroz1 commented on 1_Mechanics/02_GMGuide.md:
```suggestion
    - **Minor:**
        - *Trait*: Descrip
    - **Adversary:** 
```
